qt/qt_common/map_widget.cpp

```diff
@@ -504,7 +504,7 @@ void MapWidget::wheelEvent(QWheelEvent * e)
     return;
 
   QOpenGLWidget::wheelEvent(e);
-  m_framework.Scale(exp(e->delta() / 360.0), m2::PointD(L2D(e->x()), L2D(e->y())), false);
+  m_framework.Scale(exp(e->angleDelta().y() / 360.0), m2::PointD(e->position().x(), e->position().y()), false);
```

Comment:
Please, keep it for Qt6 PR (you can check valid changes there). It's not obvious and the scrolling speed changes.